crates/remote/src/ssh_session.rs

```diff
@@ -2112,12 +2214,30 @@ impl SshRemoteConnection {
         if !build_remote_server.contains("nocompress") {
             delegate.set_status(Some("Compressing binary"), cx);
 
-            run_cmd(Command::new("gzip").args([
-                "-9",
-                "-f",
-                &format!("target/remote_server/{}/debug/remote_server", triple),
-            ]))
-            .await?;
+            #[cfg(not(target_os = "windows"))]
+            {
+                run_cmd(Command::new("gzip").args([
+                    "-9",
+                    "-f",
+                    &format!("target/remote_server/{}/debug/remote_server", triple),
+                ]))
+                .await?;
+            }
+            #[cfg(target_os = "windows")]
+            {
+                // On Windows, we use 7z to compress the binary
```

Comment:
I understand those are dev scripts so we can be more lenient, but curious how old is 7z support in Windows?

IIRC it was added into Windows 11, last or this year?